sdk/storage/azure-storage-queue/apireview/azure/storage/queue/aio/__init__.pyi

```diff
@@ -0,0 +1,8 @@
+# Stubs for azure.storage.queue.aio (Python 3)
+#
+# NOTE: This dynamically typed stub was automatically generated by stubgen.
+
+from ..models import AccessPolicy as AccessPolicy, CorsRule as CorsRule, Logging as Logging, Metrics as Metrics, QueueMessage as QueueMessage, QueuePermissions as QueuePermissions, QueueProperties as QueueProperties, RetentionPolicy as RetentionPolicy
```

Comment:
Don't see the point to import any models in this namespace

Reply:
I was think about this and changed my opinion on it.
Having the option to import same model from two different places might be frowned upon..something like
"which one should i use?"...."how are they different"..."wait..they are not different"....."let me meme about this"

Reply:
#7864